feature/home/src/main/java/com/epmedu/animeal/home/presentation/HomeScreen.kt

```diff
@@ -22,11 +24,22 @@ fun HomeScreen() {
         initialValue = AnimealBottomSheetValue.Hidden
     )
 
-    HomeScreenUI(
-        state = state,
-        bottomSheetState = bottomSheetState,
-        onScreenEvent = viewModel::handleEvents
-    )
+    if (state.cameraState is CameraState.Enabled) {
+        CameraView(onImageCaptured = {
+            viewModel.handleEvents(HomeScreenEvent.CameraEvent.TakeNewPhoto(it))
+        }, onError = {
+
```

Comment:
Could you add something here to handle the error case? Like showing any popup, or logging the error, or adding todo.

Reply:
@Andrii-Chornyi Could you attach the link to the Figma design, and ticket description, it will makes our life easy :)

Reply:
Added disabling of Camera for this onError.
Ticket:
https://jira.epam.com/jira/browse/EPMEDU-793